src/commands/delete.ts

```diff
@@ -0,0 +1,88 @@
+import * as path from "path";
+import * as fs from "fs-extra";
+import cli from "cli-ux";
+
+import TartCommand from "../TartCommand";
+import { createExecaCommand, checkDumpName } from "../utils";
+
+export default class Load extends TartCommand {
+  static description = "delete a local database dump";
+
+  static examples = ["$ tart delete myDB", "$ tart delete myDB@1.0.0"];
+
+  static flags = {
+    ...TartCommand.flags,
+  };
+
+  static args = [
+    {
+      name: "deleteFile",
+      required: true,
+      description: "name of file to delete",
+    },
+  ];
+
+  async run() {
+    await this.runHook("beforeDelete");
+
+    const { args } = this.parse(Load);
+    const { deleteFile } = args;
+
+    if (checkDumpName(deleteFile)) {
+      await this.deleteGitTag(deleteFile);
+    } else {
+      await this.deleteLocalFile(deleteFile);
+    }
+
+    await this.runHook("afterDelete");
+  }
+
+  async deleteGitTag(deleteFile: string) {
+    const git = createExecaCommand("git", {
+      cwd: this.localConfig.repoDir,
+    });
+    const { stdout } = await git(["tag"]);
+    if (!stdout.includes(deleteFile)) {
```

Comment:
We wanted to avoid using includes like this. I changed publish to use a git command to check if the tag exists, you can also use this. We might want to make it a util.

```suggestion
    try {
      await git(["rev-parse", `${file}^{tag}`]);
    } catch (err) {
      this.error(
        `${file} is not a valid file in the repo. Try:\n   tart save ${file}@\n   tart publish ${file}@`
      );
    }
```

Reply:
Done! I added the `rev parse` way of doing it, but I kept the same messaging as before